packages/nuxt/test/nuxt.spec.ts

```diff
@@ -33,4 +33,8 @@ describe('works with nuxt', async () => {
     expect(html).not.toContain('I should not be serialized or hydrated')
     expect(html).toContain('skipHydrate-wrapped state is correct')
   })
+
+  it('throws an error server-side when the nuxt context is not available', async () => {
+    await expect($fetch('/usage-after-await')).rejects.toThrow()
```

Comment:
Let's make this a bit stricter and include part of the error message with `toThrowError()` or similar